src/main/cpp/src/get_json_object.cu

```diff
@@ -305,6 +304,8 @@ std::unique_ptr<cudf::column> get_json_object(cudf::strings_column_view const& c
   // parse the json_path into a command buffer
   auto path_commands_optional = parse_path(json_path);
 
+  auto options = json_parser_options{};
```

Comment:
As a follow on for cleanup. If we don't have a need to pass down a json_parser_options class, then do we even have a need for the class?  Would it be better to just remove it because there is only one set of options we ever run with?

Reply:
This [commit](https://github.com/NVIDIA/spark-rapids-jni/pull/1868/commits/d45422cb9cf2028a1496f5b15d03737e21c5c9dd) removed the `json_parser_options`.
When merging, will remove it also.